OpenRA.Mods.Common/Scripting/Global/TriggerGlobal.cs

```diff
@@ -291,6 +291,38 @@ public void OnAllKilledOrCaptured(Actor[] actors, LuaFunction func)
 			}
 		}
 
+		[Desc("Call a function when all of the actors in a group have been killed, captured or sold. " +
+			"The callback function will be called as func().")]
+		public void OnAllKilledCapturedOrSold(Actor[] actors, LuaFunction func)
```

Comment:
This is starting to get silly.  Please just bind the individual triggers instead, or use a variation of `OnRemovedFromWorld`.